functionnal_tests/cypress/integration/account/content_account_spec.js

```diff
@@ -276,6 +276,12 @@ describe('Account page', () => {
         cy.contains('.account__userpreference__setting__spacename', 'You are not a member of any space yet')
       })
     })
+    describe('Profile link button', () => {
+      it('should redirect to user\'s public profile', () => {
+        cy.get('.userinfo__profile_button').click()
+      cy.url().should('include', PAGES.PROFILE);
```

Comment:
Missing user id (if you want an example you can see functionnal_tests/cypress/integration/activity_feed/activity_spec.js) and be careful with indentation also.